org.kiwix.desktop.json

```diff
@@ -0,0 +1,121 @@
+{
+    "app-id": "org.kiwix.desktop",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
```

Comment:
Would be nice to switch to 5.12, it's out since 2 weeks or so.

Reply:
Changed in last forced push.